spec/features/ncr/work_orders/post_approval_modification_spec.rb

```diff
@@ -82,7 +82,10 @@ def expect_budget_approvals_restarted(work_order)
     ))
 
     approver = work_order.budget_approvers.first
+    expect(approver.email_address).to eq(Ncr::Mailboxes.ba61_tier1_budget.email_address)
     reapproval_mail = deliveries.find { |mail| mail.to.include?(approver.email_address) }
     expect(reapproval_mail.html_part.body).to include('Approve')
+    approver_page_row = page.find(:css, ".step-row.pending.position-1")
```

Comment:
I recommend looking at the Proposal Page object ( `spec/support/page_objects/proposal_page.rb`) and the tests that use it, since page objects are a better home for CSS selectors and other HTML-dependent test details.